IntelliTect.Multitool/ReleaseDateAttribute.cs

```diff
@@ -6,22 +6,17 @@ namespace IntelliTect.Multitool;
 /// <summary>
 /// The release date assembly attribute
 /// </summary>
+/// <remarks> 
```

Comment:
Curious why switch away from summary?

Reply:
This is just the way the refactoring to primary constructor transferred the comments, I noticed it but assumed that it knew what the standard for the commenting on primary constructors is going to be so I left it, thoughts?

Reply:
Seems like a bug in the refactoring IMO.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags?WT.mc_id=DT-MVP-5003472#general-tags

Reply:
I'll remove, https://github.com/IntelliTect/Multitool/pull/101/commits/cd4a0799672098817cf2462ebd29367279ca8153

If I should change it again just lmk and I'll create a patch